src/Routes/Routes.tsx

```diff
@@ -110,19 +111,15 @@ export const Routes = memo(() => {
     () =>
       appRoutes.map(route => {
         const MainComponent = route.main
-        if (isMobile && !state.isConnected) {
-          const to = {
-            pathname: '/connect-wallet',
-            search: `returnUrl=${location?.pathname ?? '/trade'}`,
-          }
```

Comment:
We don't need this anymore since mobile is automagically handled again (no explicit route allowed)